src/main/java/org/springframework/cloud/deployer/spi/mesos/chronos/ChronosTaskLauncherProperties.java

```diff
@@ -61,6 +65,11 @@
 	 */
 	private String ownerName;
 
+	/**
+	 * A set of constraints to apply to any deployed app, as a comma separated set of (field operator param?) triplets.
```

Comment:
Could be, but I chose not to, as this is really about placing an app/container (that is a "task" in our language, but...)

I'm fine with either way